Denik CZ.js

```diff
@@ -0,0 +1,110 @@
+function detectWeb(doc, url) {
+		return "newspaperArticle";
+}
+
+function doWeb(doc, url) {
+	var arts = new Array();
+	scrape(doc);
+}
+
+function scrape(doc) {
+	var newArticle = new Zotero.Item('newspaperArticle');
+	newArticle.url = doc.location.href;
+	newArticle.title = ZU.trimInternal(ZU.xpathText(doc, '//div[@class = "dv3-clanek-left-fix"]/h1')).replace(/^,/, "");
+	var date = ZU.xpathText(doc, '(//div[@class="dv3-clanek-datum"])');
+	var matched = ZU.trimInternal(date).match(/dnes/)
+	if (matched) {
+		var actualDate = new Date();
+		var actualDateStr = '';
+		actualDateStr = actualDate.getUTCFullYear() + '-' +
+    	('00' + (actualDate.getUTCMonth()+1)).slice(-2) + '-' +
+    	('00' + actualDate.getUTCDate()).slice(-2);
```

Comment:
I would suggest using `ZU.dateToISO(today).split('T')[0]`. Since this is Czech, `today` should reflect the time in Czech Republic, so `var today = new Date(); today = today.setHours(today.getHours() + 2);` (CEST, since that's 8 months out of the year and there's no great way to check for DST)